<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <title>download</title>
</head>

<body>
<div align="center"><font size="6" face="Times New Roman, Times, serif"><strong>Original data description</strong>
</font></div>
<table width="800" border="0" align="center">
    <tbody>
    <tr>
        <td><p>&nbsp;</p>
            <p><strong><font size="4" face="Times New Roman, Times, serif"><strong>NOTE</strong>: The original data
                 can be downloaded
                here.<a href="./download_all" title="download_all">Experimental data download</a></font></strong></p>
            <P style="text-align:justify"><strong><font size="4" face="Times New Roman, Times, serif">Two benchmark datasets S1 and S2 have been established in this study. S1 is used for training the first layer two classification model, S2 is used for training the second layer multi-classification model. S1 is composed of SAMP and Snon-AMP. SAMP is a data set composed of AMP sequences, and Snon-AMP is a data set composed of non-AMP sequences. The peptide sequences in SAMP were fetched form the APD3, Amper, and ADAM database. Those sequences that contained any code other than the 20 native amino acid codes were removed. To reduce homology bias and redundancy, the CD-HIT program was utilized to remove those sequences that had >90% pairwise sequence identity to any other in a same subset, and finally 3594 antibacterial peptide sequences were obtained. The Snon-AMP was built by the approach similar to previous work [5]. The peptide sequences in Snon-AMP came from UniProt (release 2020_07), and finally 3925 non-antibacterial peptide sequences were obtained. </P>
            <P style="text-align:justify"><strong><font size="4" face="Times New Roman, Times, serif">In this study, some AMP functional types contained too few peptides (less than 40) to have statistical significance, and hence were not further considered. According to their different functional types, the AMP sequences can be further classified into ten categories as showed in Table 1. The ten functional types were antibacterial peptides, antiviral peptides, antifungal peptides, antibiofilm peptides, antiparasital peptides, anti- HIV peptides, anticancer (antitumor) peptides, chemotactic peptides, anti-MRSA peptides, anti-endotoxin peptides respectively. The antibacterial peptides of unknown functions are deleted from SAMP to obtain S2. Finally, we obtained 2455 antibacterial peptide sequences in S2, of which 1311 belong to one functional attribute, 783 to two different functional attributes, 239 to three different functional attributes, 75 to four different functional attributes, 32 to five different functional attributes. </P>

        </td>
    </tr>
    </tbody>
</table>
<div align="right" style="width:50%;">
    <a href="javascript:window.close()">
        <span style="font-size:20px">
            Close
        </span>
    </a>
</div>
<div id="footer" style="font-size:15px">

    <p align="center">Xiao Lab:Bioinformatic Team (<a href="http://www.jci-bioinfo.cn/">Xiao Lab</a>)
        ----------------------------------------------------Contact: 532606318@qq.com</a></p>

</div>
</body>
</html>